crates/lib/src/error/runtime.rs

```diff
@@ -50,6 +50,11 @@ pub enum RuntimeError {
 		err: super::FsWatcherError,
 	},
 
+	/// Events from the keyboard event source
+	#[error("keyboard watcher error")]
+	#[diagnostic(code(watchexec::runtime::keyboard_watcher))]
+	KeyboardWatcher,
```

Comment:
Would it be sensible to have some kind of `kind` field on this so that we can disambiguate any future keyboard errors that might be identified without breaking builds?

Reply:
Sorry for the delay! Yeah what would be good is to do like the FsWatcher error, minus the `kind` field which isn't needed.